addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+
+1. Log in to the `kp_default_repository` locally.
+
+    ```bash
+    docker login <REGISTRY-HOSTNAME> -u <REGISTRY-USER>
+    ```
+
+   > Note: The `<REGISTRY-HOSTNAME>` must be the registry prefix for its corresponding registry
+   > - For [Dockerhub](https://hub.docker.com/) this should be `https://index.docker.io/v1/`. `kp` also offers a simplified way to create a Dockerhub secret with a `--dockerhub` flag.
+   > - For [GCR](https://cloud.google.com/container-registry/) this should be `gcr.io`. If you use GCR then the username can be `_json_key` and the password can be the JSON credentials you get from the GCP UI (under `IAM -> Service Accounts` create an account or edit an existing one and create a key with type JSON). `kp` also offers a simplified way to create a gcr secret with a `--gcr` flag.
+
+2. Create a ClusterStore.
+
+   A store resource is a repository of [buildpacks](http://buildpacks.io/) packaged
+   in [buildpackages](https://buildpacks.io/docs/buildpack-author-guide/package-a-buildpack/) that can be used by kpack
+   to build OCI images. Later in this tutorial, you will reference this store in a Builder configuration.
+
+   We recommend starting with buildpacks from the [paketo project](https://github.com/paketo-buildpacks). The example
+   below pulls in java buildpack from the paketo project. If you would like to use a different language you can select a different buildpack.
+
+    ```
+    kp clusterstore save default -b gcr.io/paketo-buildpacks/java
+    ```
+
+   > Note: Buildpacks are packaged and distributed as buildpackages which are OCI images available on a Docker registry. Buildpackages for other languages are available from [paketo](https://github.com/paketo-buildpacks).
+
+3. Create a ClusterStack.
+
+   A stack resource is the specification for
+   a [cloud native buildpacks stack](https://buildpacks.io/docs/concepts/components/stack/) used during build and in the
+   resulting app image.
+
+   We recommend starting with the [paketo base stack](https://github.com/paketo-buildpacks/stacks) as shown below:
+
+    ```
+    kp clusterstack save base --build-image paketobuildpacks/build:base-cnb --run-image paketobuildpacks/run:base-cnb
+    ```
+
+4. Create a Builder.
+
+   A Builder is the kpack configuration for a [builder image](https://buildpacks.io/docs/concepts/components/builder/)
+   that includes the stack and buildpacks needed to build an OCI image from your app source code.
+
+   The Builder configuration will write to the registry with the secret configured in step #1 and will reference the
+   stack and store created in step #3 and step #4. The builder order will determine the order in which buildpacks are
```

Comment:
Is there something missing in this?
_The builder order will determine the order in which buildpacks are used in the builder._

Reply:
I will add a link for that part of the spec